exchange/bidder.go

```diff
@@ -539,6 +539,7 @@ func (bidder *bidderAdapter) doRequestImpl(ctx context.Context, req *adapters.Re
 
 	if tmaxAdjustments != nil && tmaxAdjustments.Enabled && tmaxAdjustments.BidderResponseDurationMin > 0 {
 		if hasShorterDurationThanTmax(&bidderTmaxCtx{ctx}, *tmaxAdjustments) {
+			bidder.me.RecordTMaxTimeout()
```

Comment:
lets update `doRequestImpl` unit tests for this

Reply:
Updated in [e44b312](https://github.com/prebid/prebid-server/pull/2892/commits/e44b3125370ed31439834b14175d8b7d046c25d3)